spec/api-web-contents-spec.js

```diff
@@ -322,4 +323,208 @@ describe('webContents module', function () {
       })
     })
   })
+
+  describe('zoom api', () => {
+    const zoomScheme = remote.getGlobal('zoomScheme')
+    const hostZoomMap = {
+      'host1': 0.3,
+      'host2': 0.7,
+      'host3': 0.2
+    }
+
+    before((done) => {
+      let protocol = session.defaultSession.protocol
```

Comment:
Looks like this could be `const` along with several other variables added in these new test blocks, maybe take a pass at `const`-ing these?